lib/server/parse-websocket.js

```diff
@@ -32,5 +33,15 @@ function parseWebSocketRequest (socket, params) {
   )
   params.compact = -1 // return full peer objects (used for websocket responses)
 
+  params.ip = opts.trustProxy
+      ? socket.upgradeReq.headers['x-forwarded-for'] || socket.upgradeReq.connection.remoteAddress
+      : (socket.upgradeReq.connection.remoteAddress && socket.upgradeReq.connection.remoteAddress.replace(common.REMOVE_IPV4_MAPPED_IPV6_RE, '')) // force ipv4
```

Comment:
I was afraid that `socket.upgradeReq.connection.remoteAddress` could not be exposed in some strange webrtc implementation and we raise an error.

But in that case why not check every member? You're right I will remove it.